CHANGELOG.next.asciidoc

```diff
@@ -244,6 +244,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add ISO8601 timestamp support in syslog metricset. {issue}8716[8716] {pull}10736[10736]
 - Add more info to message logged when a duplicated symlink file is found {pull}10845[10845]
 - Add Netflow module to enrich flow events with geoip data. {pull}10877[10877]
+- Add Filebeat kubernetes module. {issue}10812[10812] 
```

Comment:
I would link the Changelog to this PR which will have all of the action, not the original issue.